src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -106,26 +107,23 @@ public static ulong CalculateIntrinsicGas(PublicEntry entry, IReleaseSpec releas
             {
                 // here is the difference between the 0 bytes and non-zero bytes cost
                 // justified by a better compression level of zero bytes
-                long txDataNonZeroGasCost = releaseSpec.IsEip2028Enabled ? GasCostOf.TxDataNonZeroEip2028 : GasCostOf.TxDataNonZero;
-                int length = entry.Data.Length;
-                for (int i = 0; i < length; i++)
-                {
+                var txDataNonZeroGasCost = releaseSpec.IsEip2028Enabled
+                    ? GasCostOf.TxDataNonZeroEip2028
+                    : GasCostOf.TxDataNonZero;
+                var length = entry.Data.Length;
+                for (var i = 0; i < length; i++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)